Tests/SwiftParserTest/ExpressionTests.swift

```diff
@@ -2295,6 +2295,46 @@ final class StatementExpressionTests: ParserTestCase {
       """,
       substructure: DeclReferenceExprSyntax(baseName: .identifier("unsafe"))
     )
+
+    assertParse(
+      """
+      func f() {
+        unsafe()
+      }
+      """,
+      substructure: DeclReferenceExprSyntax(baseName: .identifier("unsafe"))
+    )
+
+    assertParse(
+      """
+      func f() {
+        unsafe ()
```

Comment:
Technically, this is still source breaking because we parse it as a call to `unsafe` right now. But maybe it’s not too bad in practice.